zedd-app/src/components/Calendar.tsx

```diff
@@ -243,11 +288,15 @@ const CalendarBase = <T extends Interval>({
         const prevBlock = slices.find((s) => dateEqual(s.end, block.start))
         if (prevBlock) local.currentlyDragging.push({ block: prevBlock, startEnd: 'end' })
       }
+      if ('complete' === pos) {
```

Comment:
The logic from saveStartDragPosition can be added to this if block as we only need the values in that case. No explicit additional event handler is necessary.